### DISCUSSION TITLE
Osu lazer editor suggestion

### DISCUSSION THREAD

**Discussion by linManuelMirinda** [ppy/osu#16261] [Ideas] [answered]
Not sure if it's possible to be implemented well, but it would be cool if there was an option to play a song in the editor and by clicking or clicking and dragging z/x place a note/slider at the exact moment in the song. And unlike in the normal editor, it would be more like playing a map while creating patterns live. Might be easier for people like me, who want to make playable maps(for private use) with songs that were not yet mapped properly, but have no experience in mapping. It's hard to time them normally but with the song playing you could hear the beat and go along with it. Might also be helpful for making quick drafts by professional mappers idk. 

I imagine it being compatible with the normal editor. You could use it first but if you want, edit the map normally later, move objects and place them one by one. Or never use it and just map normally.

**Comment by nahkd123** [comment]
Right now you can click inside the playfield while playing to place hit circles. As for sliders, it's actually quite complicated to implement (slider velocity for example).

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/6608
